Makefile

```diff
@@ -121,42 +88,6 @@ push-tag: # example usage: make push-tag remote=origin tag=v1.100.1
 	 	git push $(remote) "$${dir:2}/$(tag)"); \
 	done
 
-DEPENDABOT_PATH=./.github/dependabot.yml
-.PHONY: gendependabot
-gendependabot: # generate dependabot.yml
```

Comment:
Is this replaced with `goyek`? Can we keep it if not?

Reply:
I will revert it